\documentclass[final]{beamer}
%% based on the Dreuw style
\mode<presentation>
{
\setbeamertemplate{block begin}{
  \vskip.25ex
  \begin{beamercolorbox}[rounded=true,shadow=true,leftskip=1cm,colsep*=.75ex]{block title}%
    \usebeamerfont*{block title}\insertblocktitle
  \end{beamercolorbox}%
  {\ifbeamercolorempty[bg]{block body}{}{\nointerlineskip\vskip-0.5pt}}%
  \usebeamerfont{block body}%
  \begin{beamercolorbox}[rounded=true,shadow=false,colsep*=.75ex,sep=.75ex,vmode]{block body}%
    \ifbeamercolorempty[bg]{block body}{\vskip-.25ex}{\vskip-.75ex}\vbox{}%
  }
  \setbeamertemplate{block end}{
  \end{beamercolorbox}
}
\setbeamertemplate{headline}{
  \leavevmode

  \begin{beamercolorbox}[wd=\paperwidth]{headline}
    \begin{columns}[T]
      \begin{column}{.02\paperwidth}
      \end{column}
      \begin{column}{.22\paperwidth}
        \begin{center}
         \includegraphics[scale=0.1]{transparentStatisticsLaura201617.jpg}
         \includegraphics[scale=0.16]{nsf.png}
        \end{center}
        \vskip0.5ex
      \end{column}
      \begin{column}{.675\paperwidth}
        \vskip4ex
        \raggedleft
        \usebeamercolor{title in headline}{\color{fg}\textbf{\LARGE{\inserttitle}}\\[1ex]}
        \usebeamercolor{author in headline}{\color{fg}\large{\insertauthor}\\[1ex]}
        \usebeamercolor{institute in headline}{\color{fg}\large{\insertinstitute}\\[1ex]}
      \end{column}
      \begin{column}{.1\paperwidth}
        \vskip1cm
        \begin{center}
          \includegraphics[scale=0.5]{Purduelogo.jpg}
        \end{center}
        \vskip1.5cm
      \end{column}
      \begin{column}{.03\paperwidth}
      \end{column}
    \end{columns}
  \end{beamercolorbox}

  \begin{beamercolorbox}[wd=\paperwidth]{lower separation line head}
    \rule{0pt}{2pt}
  \end{beamercolorbox}
}
}
\usecolortheme{purduegold}
\usepackage{times}
\usepackage{tikz}
\usepackage{pgfplots}
\pgfplotsset{compat=1.5}
\usetikzlibrary{calc}
\usepackage{amsmath,amsthm, amssymb, latexsym}
\boldmath
\usepackage[english]{babel}
\usepackage[latin1]{inputenc}
\usepackage[size=custom,width=101,height=76, scale=1,debug]{beamerposter}

\usepackage[round]{natbib}
\usepackage{graphicx}
\usepackage{url}
\usepackage{bm}

\usepackage{lipsum}  

\graphicspath{{figures/}}
  \title[Fancy Posters]{Rates of DNA Mutation in Genes and Inte-Gene Regions}
  \author{Laszlo Csonka, Mark Daniel Ward, Brian French, and Nicole Markley}
  \institute{Department of Biology, Purdue University, West Lafayette, IN}
%  \date{March 25, 2016}

  \begin{document}
  \begin{frame}{}
    \begin{columns}[t]

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

    \begin{column}{.32 \linewidth}
    \begin{block}{\large Abstract}
The aim of our analysis was to compare the rates of evolution of DNA sequences of genes and intergenic regions. The chromosomes of the related genera of bacteria, Escherichia coli K-12, Salmonella enterica serovar Typhimurium, and Citrobacter koseri, contain extensive regions in which the order of genes of identical. The fact that the gene order is so conserved suggests that both the genes and intergenic spacers were derived by evolution from corresponding sequences in the last common ancestor of the three organisms, and it enabled us to compare the sequence changes in functional genes and in intergenic spacers across the three organisms.
\newline
\newline
Pairwise comparisons using BLASTN were used to identify regions that had shared genes. ClustalW was then used to identify single base mutations in both genes and inter-gene regions. Mutation frequencies were determined for all paired comparisons. This analysis indicated that the mutation frequencies are significantly higher in intergenic spacers do not have specific sequence-dependent functions, and therefore, they can accumulate mutations more liberally than genes, which are constrained by their function.
    \end{block}

\begin{block}{\large Background}
Genomes have both useful genetic information coded in genes and presumed useless inter-gene regions termed as gaps. Gene pairs between organisms can be isolated using a tool called \textbf{B}asic \textbf{L}ocal \textbf{A}lignment \textbf{S}earch \textbf{T}ool, commonly referred to as BLAST. Local alignment can be done with an algorithm called CLUSTALw, and the results from this can be used to isolate base-level changes in sections of DNA.

\newline
\textit{Salmonella Typhimurium LT2}, \textit{Citrobacter Koseri}, and \textit{Escherchia Coli MG1655} were chosen because they are closely related members of \textit{Enterobacteriaceae}
\includegraphics[scale = 1]{phylogeny.png}
\newline
The phylogeny tree shows the relative relations of the three bacteria chosen to be analyzed. 
\end{block}

 \begin{block}{\large Motivation}
Mutation in genes has a double constrainment of DNA's built in error-handling as well as maintaining function. Gaps are suspected to have no function {\color{red} need a citation here probably}, so they should mutate faster. We suspected that choosing pairs of bacteria that were closely related could be used to show rates of mutation in homologous genes and gap regions.
\end{block}

\begin{block}{\large Question}
What degree of similarity is there between gene and gap mutation rates, and can this be used to determine if there is any functional constrainment in gaps? 
\end{block}

 \begin{block}{\large Hypothesis}
We hypothesize that there is no preservation of order shown in gap regions, so they should be substantially more random than genes.

	\end{block}

\end{column}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{column}{.32 \linewidth}

\begin{block}{\large Illustrations}
Figure 1
	\includegraphics[scale = .455]{SalECPercentage.jpeg}
	\newline
Figure 2
 \includegraphics[scale = .455]{CBSalPercentage.jpeg}
 \newline
Figure 3
 \includegraphics[scale = .455]{CBECPercentage.jpeg}


\end{block}


\begin{block}{\large Application}
  Now I can discuss why that result matters to people.
\end{block}

\end{column}



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{column}{.32 \linewidth}

\begin{block}{Procedure}
Genomes were run through BLAST to identify matching regions for each pair comparison. BLAST matches were presumed to be homologous genes. Two matching regions with a non-matching region between them led the non-matching region to be labeled as a gap. Any gap longer than 500 bases was presumed to contain a gene that wasn't present in the compared genome and was therefore discarded.
\newline
Matched regions (both genes and gaps) were run through CLUSTALw and any base mismatch preceded and followed by at least 3 matching bases was presumed to be a mutation. Mutation rates were then tabulated, plotted, and statistically tested.
\end{block}
\begin{block}{\large Conclusion}
Looking at the three figures off to the left, we have depictions of the three pairwise comparisons. The plots show the relative frequency of mutations in genes vs gaps. Gaps are shown in black, genes are in white. What immediately stands out is that the A to G/G to A and T to C/C to T mutation is clearly more frequent. Likely, this is because A and G are the two purine bases and T and C are the two pyrimidines.

Running statistical tests across the comparison, the gene and gap mutations are clearly from different distributions. Comparing the distribution of gene and gap mutations for Salmonella and E Coli with insertions and deletions left in, a chi-squared test returns a probability that they came from the same distribution of less than 2.2x10\^-16. When insertions and deletions are ignored, the likelihood that gene and gap mutations are from the same distribution rises to 1.766x10\^-13. Similar numbers arise from chi-squared testing the other two pairwise comparisons. 

This is conclusive evidence that genes and gaps have differing distributions for mutation frequencies. From this, we can conclude that there is some mechanism limiting mutations in genes. As mentioned in the background section, this is likely due to genes having to maintain function despite mutation, while gaps can mutate freely.



\end{block}


%%% If you know how to use BibTeX
%%%	  \begin{block}{\large References}
%%%	  \vspace{-.5cm} \footnotesize
%%%	\bibliographystyle{abbrvnat}     % mathematics and physical sciences
%%%	\bibliography{forbidden}   % name your BibTeX data base
%%%	\vspace{-.25cm}
%%%	 \end{block}
	
	  \begin{block}{Acknowledgments}
This research is supported by NSF grant DMS \#1246818.
	 \end{block}



        \end{column}

    \end{columns}

  \end{frame}
\end{document}	



